ViAn/Test/test_report_generator.cpp

```diff
@@ -0,0 +1,104 @@
+#include "test_report_generator.h"
+
+/**
+ * @brief test_report_generator::test_report_generator
+ * @param parent
+ */
+test_report_generator::test_report_generator(QObject *parent) : QObject(parent) {
+    this->file_handler = new FileHandler();
+}
+
+/**
+ * @brief test_report_generator::test_init
+ * This will initalize everything that is needed for testing the report generator.
+ */
+void test_report_generator::test_init()
+{
+    this->proj1 = file_handler->create_project("TEST_PROJ1");
+    this->proj2 = file_handler->create_project("TEST_PROJ2");
+
+    std::string video1_path = "C:/Users/georg/Desktop/Pumparna.avi";
+    std::string video2_path = "C:/Users/georg/Desktop/England4k.mov";
+    QString dir_path1 = "C:/TEST_PROJ1/Bookmarks";
+    QString dir_path2 = "C:/TEST_PROJ2/Bookmarks";
+    QString bookmark1_path = "C:/Users/georg/Desktop/england_bild.jpg";
```

Comment:
yes, or you have to change the paths to a image or other movies. I dont know how to solve this otherwise. these tests are a bit special. Any idea?